<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>browser</class>
 <widget class="QMainWindow" name="browser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>神龙翻译 - 网页</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>翻译引擎：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="baidu">
        <property name="text">
         <string>百度</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="youdao">
        <property name="text">
         <string>有道</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="bing">
        <property name="text">
         <string>必应</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="google">
        <property name="text">
         <string>谷歌</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ciba">
        <property name="text">
         <string>词霸</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="oxford">
        <property name="text">
         <string>牛津</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QRadioButton" name="tencent">
        <property name="text">
         <string>腾讯</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="auto_update">
        <property name="text">
         <string>实时更新</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="web_layout"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
